src/pages/contributing/translation-program/contributors.tsx

```diff
@@ -66,6 +62,7 @@ const Contributors = () => {
 
   // TODO: Remove specific user checks once Acolad has updated their usernames
   const translatorData =
+    // @ts-expect-error Not able to manually type `allTimeData` (too big). Need to generate a type signature.
```

Comment:
@pettinarip This is failing Netlify deploy, where Typescript is assuming there is no error when there actually is.

To get rid of this and to not error on an `any` type, I would try and create a type for the `allTimeData` json because an inference is not available. There is a generator available to copy/paste the json to spit out a type set. Then can apply it here.

Reply:
See #12037

Reply:
Merged!